DEVELOPING.md

```diff
@@ -143,7 +143,12 @@ cd docroot
 To run all tests for a particular module:
 ```
 cd docroot
-../vendor/bin/phpunit -c core --group MODULE --debug
```

Comment:
Hey @panshulK I think this code also needs to be changed:
https://github.com/acquia/acquia_cms/pull/873/files#diff-8822179a8da757fde968cf80a9162e1c1d6f8cc170509cfbdea10be03b7e7d20R141 to

`../vendor/bin/phpunit -c core profiles/contrib/acquia_cms/modules --debug`